README.md

```diff
@@ -26,22 +26,40 @@ Renders the gizmo for a single frame
 
 Creates a new object which can be rendered over multiple frames
 
-#### object:enable() -> void
+### gizmo.\<type\>.createWithStyle(optional_style, ...) -> object
+
+Creates a new object and applies the style
+```ts
+const arrow = Gizmo.arrow.createWithStyle({
+  color: new Color3(1, 0, 0),
+  scale: 2
+}, Vector3.zero, Vector3.one.mul(15));
+```
+
+#### object.enable() => void
 
 Starts rendering the gizmo
 
-#### object:disable() -> void
+#### object.disable() => void
 
 Stops rendering the gizmo
 
-#### object:update(...) -> void
+#### object.update(...) => void
 
 Updates the gizmos appearance
 
 #### object.style
```

Comment:
I'm happy with setStyle, it makes the ergonomics of setting easier as we can just default to the main globalStyle object. Could we actually remove direct access to the style object? We'd need a setGlobalStyle function on the exports, but I don't have an issue with this. We'd also likely want to add a corresponding getStyle function for completion sake.

I think it makes more sense to go this road than having the ability to set the style manually (which doesn't fill), and the setStyle function because it's two different ways to do similar things that aren't quite the same.

Reply:
yeah i will add that